Lib/test/test_typing.py

```diff
@@ -104,13 +98,9 @@ def test_errors(self):
         with self.assertRaises(TypeError):
             Any[int]  # Any is not a generic type.
 
-    def test_cannot_subclass(self):
-        with self.assertRaises(TypeError):
-            class A(Any):
-                pass
-        with self.assertRaises(TypeError):
-            class A(type(Any)):
-                pass
+    def test_can_subclass(self):
+        class X(Any): pass
+        X()
```

Comment:
Should assert that X is a subclass of Any and X() is an instance. And also assert that an unrelated class *isn't* an instance/subclass of Any. Also good to do some tests involving multiple inheritance.